src/UnbxdSearchWrapper.js

```diff
@@ -173,6 +173,9 @@ class UnbxdSearchWrapper extends Component {
             typeof onPageLoad == 'function' &&
                 onPageLoad(unbxdCore.getResponseObj());
         }
+        if (unbxdCore.getResponseObj() === null && queryParams[unbxdCore.options.searchQueryParam]) {
```

Comment:
Did you test this change on category page refresh?

Reply:
yes

Reply:
Try these steps to confirm if category page refresh is working.

1. open the network tab
2. navigate to one of the category pages.
3. look at the api. It should contain -- category?q=categoryPath.....
4. refresh the page.
5. look at the new api. It should still contain -- category?q=categoryPath....

Reply:
had done already

Reply:
I can see that category pages refresh breaks. Please follow the steps mentioned.

Reply:
lets get on a call to discuss this, as per the steps mentioned I am able to see the results